pkg/generated/bindata.go

```diff
@@ -2,9 +2,7 @@
 // sources:
 // assets/controller.yaml
 // assets/controller_sa.yaml
-// assets/credentials.yaml
 // assets/csidriver.yaml
-// assets/namespace.yaml
```

Comment:
Perhaps did you forget `make update`?

Reply:
no... why? we don't have these assets anymore, so these lines should be removed, right?
I tried to do `make update` again on this patch and nothing happened

Reply:
hm, I think _I_ forgot to `make update` in my first PR